heatintro.md

```diff
@@ -1,85 +1,142 @@
+
+---
+## OS::Heat::ScalingPolicy
+Properties:
+  - **change**: a number that has an effect based on change_type.
+  - **change_type:** (meaning of "change")
+  	- "change_in_capacity"
+    - "percentage_change_in_capacity", 
+    - "exact_capacity" 
+  - **cooldown:** minimum amount of time (in seconds) between allowable executions of this policy.
+  - **group_id:** ID of the group that this policy will affect
+  - **name:** Convenient name
+
+Attributes: (outputs)
+  - **alarm_url**: the url which triggers the policy
+
+Example:
+
+    policy_for_autoscaling_group:
+      type: OS::Heat::ScalingPolicy
+      properties:
+        adjustment_type: change_in_capacity
+        auto_scaling_group_id: { get_resource: group }
+        cooldown: 60
+        scaling_adjustment: -1
+
+---
+
+## OS::Monasca::Notification
+Properties:
+  - **address**: Address of the notification. 
         - email address
         - url
         - service key based on notification type.
-    - type: Type of the notification.
+  - **type**: Type of the notification.
         - email
         - webhook
         - pagerduty
-exameple:
-  up_notification:
-    type: OS::Monasca::Notification
+     
+Example:
+
+	notification_object:
+      type: OS::Monasca::Notification
+      properties:
+        type: webhook  # we want the notification to trigger the policy via a url
+        address: {get_attr: [some_policy, alarm_url]} # this get the alarm_url to trigger the policy
+
+---
+
+## **OS::Monasca::AlarmDefinition**
+Properties:
+- **expression**: Expression of the alarm to evaluate.
+	
+    eg: `avg(cpu.utilization_perc{group_by=something}) > 50 times 3`
+    
+Optional Properties
+- **alarm_actions**
+- **ok_actions**
+- **undetermined_actions**
+- **actions_enabled**
+- **match_by**
+- **severity**
+- description
+- name
+
+Example:
+
+	fire_alarm:
+    type: OS::Monasca::AlarmDefinition
     properties:
-      type: webhook
-      address: {get_attr: [scale_up_policy, alarm_url]}
-
-
-*OS::Monasca::AlarmDefinition*
-    Properties:
-    - expression
-    Expression of the alarm to evaluate.
-    String value expected.
-    Updates cause replacement.
-
-    Optional Properties
-
-    - actions_enabled
-    - alarm_actions
-        - List of Monasca notification.
-    - description
-    - name
-    - match_by: The metric dimensions to match to the alarm dimensions
-    - ok_actions
-    - severity
-    - undetermined_actions: The notification methods to use when an alarm state is UNDETERMINED.
-        - List of Monasca notification.
+      ....
+      expression:
+        str_replace:
+          template: avg(cpu.utilization_perc{scale_group=group_by}) > 50 times 3
+          params:
+            group_by: {get_param: "OS::stack_id"} # use stack_id to group instances
+      severity: high
+      alarm_actions:
+        - {get_resource: notification_object_from_the_stack }
+---
+    
+## Big Picture
+
+	AlarmDefinition >>> has
+  		Notification   >>> triggers
```

Comment:
If you are trying to show the cascade of objects, you might pull Notification line to the left.